app/unit-tests/src/org/commcare/update/UpdateWorkerTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.commcare.update
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.commcare.CommCareTestApplication
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@Config(application = CommCareTestApplication::class)
+@RunWith(AndroidJUnit4::class)
+class UpdateWorkerTest {
```

Comment:
Ohh Didn't realise you're adding tests for worker. Awesome!

Reply:
lol yeah, started with that, ended up with something else